src/Roadkill.Web/Views/Pages/Edit.cshtml

```diff
@@ -1,21 +1,213 @@
 ﻿@model PageViewModel
 @{
 	ViewBag.Title = SiteStrings.EditPage_Title;
+    string myGuid = Guid.NewGuid().ToString();
 }
 
+<script src="~/node_modules/markdown-it/dist/markdown-it.min.js?v=@(myGuid)"></script>

```

Comment:
You need to use a `~dist` folder or similar really, not reference node_modules as it shouldn't be in the out folder